index.js

```diff
@@ -85,7 +88,31 @@ app.get('/api/v1/favorites/:id', cors(corsOptions), (request, response) => {
   .catch(error => {
     response.status(400).json({message: 'favorite not found'});
   });
-})
+});
+
+app.get('/api/v1/playlists', cors(corsOptions), (request, response) => {
+  database('playlists')
+  .join('playlist_songs', {'playlists.id': 'playlist_songs.playlist_id'})
+  .join('songs',{'songs.id': 'playlist_songs.song_id'})
+  .select([
+    'playlists.id as id',
+    'playlists.name as name',
+    database.raw("JSON_AGG(songs) as favorites")
+  ])
```

Comment:
I would be curious as to what line 100 is. Looks like some kind of aggregation function?